pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -872,12 +898,39 @@ pub mod pallet {
 		pub fn set_season(
 			origin: OriginFor<T>,
 			season_id: SeasonId,
-			season: SeasonOf<T>,
+			season: Option<SeasonOf<T>>,
+			season_meta: Option<SeasonMeta>,
+			season_schedule: Option<SeasonScheduleOf<T>>,
+			trade_filters: Option<TradeFilters>,
```

Comment:
Same as above, I think from a separation of concern perspective, doesn't it make more sense to have separate xts? 

Of have a `set_season` that has mandatory fields and a `update_xxx` to change a specific value?